NEL.MESH/Services/Foundations/Mesh/MeshService.cs

```diff
@@ -70,7 +70,23 @@ public MeshService(IMeshBroker meshBroker)
             {
                 ValidateMeshMessageOnSendFile(message, authorizationToken);
 
-                HttpResponseMessage responseFileMessage = await this.meshBroker.SendFileAsync(
+                string chunkRange = GetKeyStringValue("Mex-Chunk-Range", message.Headers)
```

Comment:
Mex-Chunk-Range is an optional header. Do a check here to see if the value is string.empty. If it is empty, then set the value to "1" as we can then assume that it is a single part message.